test/Microsoft.NET.Build.Tests/GivenThatWeWantToGenerateGlobalUsings_DotNet.cs

```diff
@@ -103,6 +103,7 @@ public void It_can_generate_custom_usings()
     <Using Include=""CustomNamespace"" />
     <Using Include=""TestStaticNamespace"" Static=""True"" />
     <Using Include=""System.Biology"" Alias=""AppliedChemistry"" />
+    <Using Include=""(System.String X, int Y)"" Alias=""MyTuple"" />
```

Comment:
Let's add a direct keyword example, like making an alias for `int` or `dynamic`.